pkg/driver/driver.go

```diff
@@ -886,6 +886,15 @@ func (d Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotReque
 		return nil, missingAttr("CreateSnapshot", req.GetName(), "Name")
 	}
 
+	d.log.WithField("req.parameters", req.GetParameters()).WithField("req.secrets", req.GetSecrets()).Debug("parsing request")
```

Comment:
Are these the secrets themselves? Perhaps best not to log them :smile:

Reply:
Oops. Also logged again a bit below when logging the parsed parameters. Removed the secrets here and added a `Stringer` impl for the parameters that replaces the secret values with `***`.